google/cloud/aiplatform/experimental/vertex_model/base.py

```diff
@@ -168,10 +168,11 @@ class VertexModel(metaclass=abc.ABCMeta):
 
     """ Parent class that users can extend to use the Vertex AI SDK """
 
-    def __init__(self):
+    def __init__(self, *args, **kwargs):
         # Default to local training on creation, at least for this prototype.
```

Comment:
This should include a docstring stating the expectation that subclasses should pass their constructor arguments into this base class.